src/main/webapp/i18n/de/textAssessment.json

```diff
@@ -49,9 +49,10 @@
             "feedbackEditor": {
                 "dismissFeedback": "Möchtest du das Feedback löschen?",
                 "dismissFeedbackConfirmation": "Bitte bestätige, dass du das Feedback löschen möchtest!",
-                "feedbackCommentPlaceholder": "Gib deine Anmerkung bitte hier ein.."
+                "feedbackCommentPlaceholder": "Gib deine Anmerkung bitte hier ein..",
+                "impactWarning": "Diese Feedback kann <strong style=\"color:blue\">{{affectedSubmissionsCount}}</strong> weitere ähnliche Einreichung/en beinflussen.",
```

Comment:
Really small comment: In the German translation it should say "Dieses Feedback" instead of "diese Feedback" i

Reply:
```suggestion
                "impactWarning": "Dieses Feedback kann <strong style=\"color:blue\">{{affectedSubmissionsCount}}</strong> weitere ähnliche Einreichung/en beinflussen.",
```